addons/mail/static/src/components/follow_button/follow_button.scss

```diff
@@ -1,31 +1,3 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_FollowButton {
-    display: flex;
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
-.o_FollowButton_follow {
-    color: gray('600');
-}
-
-.o_FollowButton_unfollow {
-    color: gray('600');
-
-    &.o-following {
-        color: o-text-color('success');
-    }
-
-    &.o-unfollow {
-        color: o-text-color('warning')
-    }
-
-    &:focus {
-        background-color: gray('200');
-    }
+.o_FollowButton_unfollow:focus {
+    background-color: $gray-200;
```

Comment:
Just taking the opportunity to say that I'm not a big fan of these `mail` buttons that look inconsistently compared to the rest of the UI :/
@seb-odoo, I suggest to address it once the SCSS optimisation is done.

https://user-images.githubusercontent.com/8558351/162156326-80763dd0-af07-4ef5-9ee4-0e4ff814e5f8.mov

Reply:
Yes please, I'm all for a consistent interface (especially when the consistency is driving the quality up, and hopefully not down because of compromises), but it should not be done as part of this refactoring otherwise it will be too complex to review the changes.